caesars-palace/blocks/stats/stats.js

```diff
@@ -1,8 +1,63 @@
 export default function decorate(block) {
-  const statsContainerElement = [...block.children][0];
+  const statsRows = [...block.children];
 
-  if (statsContainerElement) {
-    const columnsInRow = statsContainerElement.children.length;
-    statsContainerElement.style = `grid-template-columns: repeat(${columnsInRow}, 1fr)`;
+  if (statsRows.length === 1) {
+    // row of stats can be short, so adjusting styling depending on columns count
+    statsRows[0].classList.add('short-stats');
+    const columnsInRow = [...statsRows[0].children].length;
+
+    const mediaSmallWidthQueryMatcher = window.matchMedia('only screen and (max-width: 767px)');
```

Comment:
The media watchers are here to have nicely centered grids, even if there are less columns to show in a line than configured in CSS